src/pages/signin/SignInModal.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import SignInPage from './SignInPage';
+import ScreenWrapper from '../../components/ScreenWrapper';
+import HeaderWithBackButton from '../../components/HeaderWithBackButton';
+import Navigation from '../../libs/Navigation/Navigation';
+import styles from '../../styles/styles';
+
+const propTypes = {};
+
+const defaultProps = {};
+
+function SignInModal() {
+    return (
+        <ScreenWrapper
+            style={[styles.signInModal]}
```

Comment:
We don't need to create a new style for this. We can reuse `styles.highlightBG`.

Reply:
Thanks for the idea! Didn't saw it while coding